src/components/screen/screen-list.jsx

```diff
@@ -171,6 +173,19 @@ function ScreenList() {
     setPage(pageNumber);
   }
 
+  /**
+   * Sets created by filter.
+   *
+   * @param {number} createdByInput - The created by filter.
+   */
+  function onCreatedByFilter(createdByInput) {
```

Comment:
Could this function take context as param, and then be exported to a common file, instead of being redeclared (copy-paste) in a lot of files.